javascripts/govuk/analytics/external-link-tracker.js

```diff
@@ -21,6 +21,15 @@
         options.label = linkText
       }
 
+      // This custom dimension will be used to duplicate the url information
+      // that we normally send in an "event action". This will be used to join
+      // up with a scheduled custom upload called "External Link Status".
+      // We can only join uploads on custom dimensions, not on `event actions`
+      // where we normally add the url info.
+      var googleAnalyticsExternalLinkUploadDimension = 36
+      var urlDimensionToJoinUploadOn = href
+
+      GOVUK.analytics.setDimension(googleAnalyticsExternalLinkUploadDimension, urlDimensionToJoinUploadOn)
```

Comment:
It might be better to make the dimension an option that can be specified here:
https://github.com/alphagov/static/blob/master/app/assets/javascripts/analytics/static-analytics.js#L27

Reply:
I've created two new PRs for this here: 

https://github.com/alphagov/govuk_frontend_toolkit/pull/439
https://github.com/alphagov/static/pull/1192